powerauth-webflow-client/src/main/webapp/WEB-INF/jboss-deployment-structure.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<jboss-deployment-structure xmlns="urn:jboss:deployment-structure:1.2">
+	<deployment>
+		<exclude-subsystems>
+			<!-- disable the resource-adapters subsystem to prevent the application's HSQLDB driver
+			from being included in application server JDBC drivers -->
+            <subsystem name="resource-adapters" />
```

Comment:
The formatting seems a bit off.

Reply:
Fixed.

The reason: I am using space chars only in files. For XML I've reconfigured IDEA to use tab char.